src/RuleManager.js

```diff
@@ -47,11 +47,9 @@ const createTestFn = ({
     matchers.push(createRootLocalNameMatcher(rootLocalName));
   }
   if (rootAttributes) {
-    const attributeMatchers = map(
-      createRootAttributeMatcher,
-      rootAttributes,
-    );
-    matchers.push(...attributeMatchers);
+    for (const name of Object.keys(rootAttributes)) {
```

Comment:
This could be written like the following to remove the need to disable the `no-restricted-syntax` ESLint rule:

```js
Object.keys(rootAttributes).forEach((name) => {
```